### PR TITLE
infoschema: improve performance

### PR DESCRIPTION
**Pull request by xhebox**
Signed-off-by: xhe <xw897002528@gmail.com>

### What problem does this PR solve?

Problem Summary: Reported and suggested by @sticnarf , `GetLatest` is on hotpath, and metrics caused some performance degradation in 5.1. This PR tries to init the counter first.

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

- No release note

### CODE REVIEW

File: infoschema/cache.go [hunk 1]
```diff
@@ -40,9 +50,9 @@ func NewCache(capcity int) *InfoCache {
 func (h *InfoCache) GetLatest() InfoSchema {
 	h.mu.RLock()
 	defer h.mu.RUnlock()
-	metrics.InfoCacheCounters.WithLabelValues("get", "latest").Inc()
+	getLatestCounter.Inc()
```
Comment (nolouch) [inline infoschema/cache.go:53]: How much performance degradation will WithLabelValues bring?

**Review by nolouch — APPROVED** [review]
lgtm

File: infoschema/cache.go (hunk 1 above)
Comment (xhebox) [reply infoschema/cache.go:53]: Several seconds, though it is not a big deal in the total performance degradation time. The degradation disappears after removing `metrics`.

**Review by nolouch — APPROVED** [review]

**Review by sticnarf — APPROVED** [review]
LGTM

**Review by JmPotato — APPROVED** [review]

**Review by morgo — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Pull request merged**